src/pages/Calculator.tsx

```diff
@@ -243,7 +258,9 @@ const Calculator: FC<RouteComponentProps> = () => {
           <h2>
             Suma: <span>{(points.total ?? 0).toFixed()}</span>
           </h2>
-          <button onClick={resetForm}>Resetuj wynik</button>
+          <button type="button" onClick={resetForm}>
```

Comment:
I remember there was some issue here. But maybe we can use `reset` type?

Reply:
Es-lint does not allow to 'reset' type - and I've found some discussions about this. Shortly - many people think that 'reset' type shouldn't be used. We can discuss this, but now I will leave it with ``type="button"``